packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/assetExplorer/assetTable/assetTableNameLink.tsx

```diff
@@ -20,7 +22,13 @@ export function AssetTableNameLink({
         updateParentAssetId(assetId);
       }}
     >
-      {assetName}
+      <span
+        data-testid='asset-name-link'
+        title={`View children assets of ${assetName}`}
```

Comment:
Is this the type of tool tip UX wanted? We do have a tool tip component that we use elsewhere in dashboard, like the one used in the component library.

Reply:
@nadeswal can you please confirm, Is it fine to go with the default tooltip as shown in the above video? Or can we add the tooltip like what we used for the component library?

Reply:
@jmbuss I tried with a` <Tooltip /> `component but it was not working as expected, outside layouts are overlaying the tooltip content like in the below screenshot,
![image](https://github.com/awslabs/iot-app-kit/assets/139960352/7a19c1ca-eae9-4e69-a1dd-1bd33c5d2272)

I tried to use an external tooltip library(https://www.npmjs.com/package/react-tooltip?activeTab=readme) and we are able to show a tooltip like the one below screenshot,
![image](https://github.com/awslabs/iot-app-kit/assets/139960352/fdf86087-bee4-4672-9bd5-3f644bdb12f6)

Since we are not using the sibling nodes to the root div(React Portals) for tooltips it causing the Z-index issues.

Reply:
I see, yes it looks like a limitation of the Tooltip component. We will need to update that component to be able to handle cases such as this. We can move forward with the title attribute now to unblock and I will create a ticket to fix the Tooltip component.